dds/DCPS/transport/rtps_udp/RtpsUdpSendStrategy.cpp

```diff
@@ -716,12 +722,12 @@ RtpsUdpSendStrategy::stop_i()
 size_t RtpsUdpSendStrategy::max_message_size() const
 {
   // TODO: Make this conditional on if the message actually needs to do this.
-  return max_message_size_
+  return max_message_size_ ? max_message_size_
```

Comment:
When would `max_message_size_` be 0?

Reply:
I'm not completely sure, but there are checks for it in `send` and `get_packet_elems_from_queue`.

Reply:
If those are in the framework, that would be for other transport types.

Reply:
Yea they are. It's probably for TCP. If that's the case I can remove it.